src/vue-virtual-table.vue

```diff
@@ -1233,11 +1233,11 @@ export default {
       self.sortParam.col = val;
       self.sortParam.direction = direction;
       let isNumber = false;
-      self.dataTemp.some((v, i) => {
```

Comment:
Here is to find a real number. If there is, this column will be regarded as number.